code/datums/syndicate_buylist.dm

```diff
@@ -124,10 +124,10 @@ proc/build_syndi_buylist_cache()
 	blockedmode = list(/datum/game_mode/revolution)
 
 /datum/syndicate_buylist/generic/chamsuit
-	name = "Chameleon Jumpsuit"
-	item = /obj/item/clothing/under/chameleon
+	name = "Chameleon Outfit"
+	item = /obj/item/storage/backpack/chameleon
 	cost = 1
```

Comment:
this is probably worth more than 1 now but i'm fine with leaving it so we get to watch more people try it out (not a request for changes)